src/SkillService/Startup.cs

```diff
@@ -182,14 +184,34 @@ private void ConfigureMassTransit(IServiceCollection services)
             host.Password(password);
           });
 
+          ConfigureEndpoints(context, cfg, _rabbitMqConfig);
         });
 
+        ConfigureConsumers(x);
+
         x.AddRequestClients(_rabbitMqConfig);
       });
 
       services.AddMassTransitHostedService();
     }
 
+    private void ConfigureEndpoints(
+      IBusRegistrationContext context, 
+      IRabbitMqBusFactoryConfigurator cfg, 
+      RabbitMqConfig rabbitMqConfig)
+    {
+      //TODO: Check proper endpoint
+      cfg.ReceiveEndpoint(rabbitMqConfig.CheckUserRightsEndpoint, ep =>
```

Comment:
wrong endpoint
add endpoint to appsettings.json and call it throw RabbitMqConfig